it/uk/gov/hmrc/agentrelationships/controllers/RelationshipsControllerISpec.scala

```diff
@@ -483,124 +479,125 @@ class RelationshipISpec extends UnitSpec
 
     //UNHAPPY PATHS
 
-    "return 404 when credentials are not found in gg" in {
-      givenAgentCredentialsAreNotFoundFor(Arn(arn))
-      givenAgentCodeIsFoundFor("foo", "bar")
+    "return 404 when credentials are not found in es" in {
+      givenPrincipalGroupIdNotExistsFor(arn)
+      givenGroupInfo("foo", "bar")
```

Comment:
`givenGroupInfo("foo", "bar")` stub already called in `givenAgentIsAllocatedAndAssignedToClient(_, "bar")`